fili-core/src/main/java/com/yahoo/bard/webservice/web/util/PaginationParameters.java

```diff
@@ -114,11 +122,37 @@ private void validate(int parameter, String parameterName) throws BadPaginationE
         }
     }
 
+    /**
+     * Verifies that the page is positive or is -1 (last page).
```

Comment:
Rather than saying "or is -1" we should say "or is the last page."

As far as this method is concerned, it shouldn't really care what value we use to represent the last page.